main.py

```diff
@@ -0,0 +1,5 @@
+from rich.console import Console
+
+if __name__ == '__main__':
+    console = Console()
+    console.print("Hello", "World!", style="bold red")
```

Comment:
Ok, so we use `rich` as 'framework'.
1. Is there any reason, why we do not use direct `print` import from rich, when we do not require to use whole `Console` API?
2. Is there any reason, why use `"Hello"` and `"World"` as two parameters inside `print` method instead of simple `"Hello, World"`?
3. Missing empty line at end of the file.

```suggestion
from rich import print

if __name__ == '__main__':
    print("[bold red]Hello, World![/]")

```

Reply:
1. According to [rich documentation](https://rich.readthedocs.io/en/latest/console.html),Console class has useful method that can help us formatting JSON. `console.print_json('[false, true, null, "foo"]')`
2. There is no reason, to fix
3. To fix

Reply:
Ad. 1 - but we are not formatting JSON here. It is just plain text.